nav2_navfn_planner/include/nav2_navfn_planner/navfn_planner.hpp

```diff
@@ -28,13 +29,14 @@
 #include "nav2_msgs/msg/costmap.hpp"
 #include "nav2_msgs/msg/path.hpp"
 #include "nav2_navfn_planner/navfn.hpp"
-#include "nav2_util/costmap_service_client.hpp"
 #include "nav2_util/robot_utils.hpp"
 #include "nav2_util/simple_action_server.hpp"
 #include "nav_msgs/msg/path.hpp"
 #include "visualization_msgs/msg/marker.hpp"
 #include "tf2_ros/transform_listener.h"
 #include "tf2_ros/create_timer_ros.h"
+#include "nav2_costmap_2d/costmap_2d_ros.hpp"
+#include "nav2_costmap_2d/cost_values.hpp"
```

Comment:
Looks like we don't use cost values here so this include can be removed.

Reply:
It is used
```
   costmap_->setCost(mx, my, nav2_costmap_2d::FREE_SPACE); 
```

Reply:
I meant we can move the include to the .cpp, since the values are not needed on the header.

Reply:
oh, yeah sure no problem